src/Components/App/App.js

```diff
@@ -1,26 +1,25 @@
 import React, { Component } from 'react';
 import { Route, Link } from 'react-router-dom';
-// import {useSelector, useDispatch} from 'react-redux';
-// import { getNYTList } from '../../actions';
 import Search from '../Search/Search';
 import BookContainer from '../BookContainer/BookContainer';
 import HaveRead from '../HaveRead/HaveRead';
 import ToRead from '../ToRead/ToRead';
 import GoalForm from '../GoalForm/GoalForm';
-// import { connect } from 'react-redux';
-// import { getBooks } from '../../apiCalls.js';
+import { connect } from 'react-redux';
 import './App.css';
 
 class App extends Component {
-  // constructor(props) {
-  //   super(props);
-  // }
+  constructor(props) {
```

Comment:
Great job rendering the user goal on App. I like that it will persist throughout all components

Reply:
This is great, I was not thrilled about the possibility that it might need to be only in the userGoal element